src/poll-itc.js

```diff
@@ -31,6 +31,8 @@ function checkAppStatus() {
             console.log(stderr)
         }
     })
+
+    exec("find /tmp -maxdepth 1 -name 'spaceship*.log' -mtime +7 -delete")
```

Comment:
Did you mean this instead?

```suggestion
    exec("find /tmp/ -maxdepth 1 -name 'spaceship*.log' -mtime +7 -delete")
```

Without the trailing `/`, it wouldn't find anything